shared/src/store/action-reducers/vehicle.ts

```diff
@@ -308,6 +310,25 @@ export namespace VehicleActionReducers {
                     });
                 }
             }
+            if (vehicle.imagesPatientsLoaded !== undefined) {
+                const numberOfPatientsInVehicle = Object.keys(
+                    vehicle.patientIds
+                ).length;
+                if (
+                    vehicle.imagesPatientsLoaded[numberOfPatientsInVehicle] !==
+                    undefined
+                ) {
+                    // TODO: why needing "!"" at the end?
```

Comment:
Because TS has no idea that this element exists. It can't relate the check above to here. You can use an extra `const` for this.